packages/page-pattern-modal/src/styles/page-pattern-modal.scss

```diff
@@ -53,15 +65,20 @@ $sidebar-width-desktop: 324px;
 	.components-modal__content {
 		overflow-y: scroll;
 		-webkit-overflow-scrolling: touch;
-		// prevents a 1px strip of content being visible after scrolling
-		margin-top: -1px;
+		margin-top: 60px;
```

Comment:
It seems removing this has about the same effect for me as adding the `60px` rule.